dist/templates/index.tmpl

```diff
@@ -0,0 +1,280 @@
+      origin = location.origin || (window.location.protocol + "//" + window.location.hostname + (window.location.port ? ':' + window.location.port: '')),
+      baseUrl = origin + location.href.substr(origin.length).replace(rUrl, indexFile),
+      headEl = document.getElementsByTagName('head')[0],
+      sync = true;
+
+      addTag('base', {href: baseUrl});
+      addTag('link', {rel: 'stylesheet', href: 'css/styles.css', type: 'text/css'});
+
+      <% _.forEach(styles, function(url) { %>
+       addTag('link', {rel: 'stylesheet', href: '<%= url %>', type: 'text/css'}, sync);
+       <% }); %>
+
+      <% _.forEach(scripts, function(url) { %>
+        addTag('script', {src: '<%= url %>'}, sync);
+        <% }); %>
+      <% if(!deferLoad) { %>
+        addTag('script', {src: 'js/scripts.min.js'}, sync);
+        addTag('script', {src: 'js/docs-setup.js'}, sync);
+      <% } %>
+      function addTag(name, attributes, sync) {
+        var el = document.createElement(name),
+        attrName;
+
+        for (attrName in attributes) {
+          el.setAttribute(attrName, attributes[attrName]);
+        }
+
+        sync ? document.write(outerHTML(el)) : headEl.appendChild(el);
+      }
+
+      function outerHTML(node){
+        // if IE, Chrome take the internal method otherwise build one
+        return node.outerHTML || (
+          function(n){
+            var div = document.createElement('div'), h;
+            div.appendChild(n);
+            h = div.innerHTML;
+            div = null;
+            return h;
+          })(node);
+        }
+      })();
+      <% if (analytics) { %>
+    // GA asynchronous tracker
+    var _gaq = _gaq || [];
+    _gaq.push(['_setAccount', '<%= analytics.account %>']);
+    _gaq.push(['_setDomainName', '<%= analytics.domainName %>']);
+
+    (function() {
+      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
+      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
+      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
+    })();
+    <% } %>
+    </script>
+  </head>
+
+  <body>
+    <header class="header">
+      <div class="navbar navbar-default navbar-fixed-top">
+        <!-- Brand and toggle get grouped for better mobile display -->
+        
+        <div class="container">
+          
+          <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" ng-click="nav.isCollapsed = !nav.isCollapsed">
+              <span class="sr-only">Toggle navigation</span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+              <span class="icon-bar"></span>
+            </button>           
+            <% if (image) { %>
+                <% if (imageLink) { %>  <a class="navbar-brand" href="<%= imageLink %>"> <% } %>
+                  <img alt="Brand" src="<%= image %>">
```

Comment:
Regarding the `dist` folder, it serves as namespace for minified files. The current workflow produces a lot of unminified files (css and js), and of course, this leads to a increased use of bandwidth for the end-user. The `dist` simply does that, instead of calling each file from the `src` folder, the app now uses the minified version. I know is not the best solución but the current workflow does not allow me to do any further changes without refactoring a lot of the functionality.

Also, I added a Gulp.js file for that. It copies the files from `src`, minifies the `css` and `js` files, contacts them and puts them in the `dist` folder for the app to get them from there.